es-design-system/pages/atoms/spacing.vue

```diff
@@ -1,75 +1,114 @@
+        </h1>
+        <p>
+            Extended from <b-link href="https://getbootstrap.com/docs/4.6/utilities/spacing/">
+                bootstrap spacing
+            </b-link>
+        </p>
+
+        <div class="mb-450">
+            <p>
+                Each spacing class is named for the percentage of the base size (16px) that generates the associated
+                pixel value.
+            </p>
+            <p>
+                For example, 32px is 200% of 16px, so the classes are named <code>p-200 m-200</code>.
+            </p>
+            <table class="table table-bordered">
+                <thead>
+                    <tr>
+                        <th scope="col">
+                            Class Name
+                        </th>
+                        <th scope="col">
+                            Multiplier (rem)
+                        </th>
+                        <th scope="col">
+                            Spacing (px)
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr
+                        v-for="space in spacers"
+                        :key="space.alias">
+                        <td>
+                            <code>{{ space.alias }}</code>
+                        </td>
+                        <td>
+                            {{ space.em }}
+                        </td>
+                        <td>
+                            {{ space.px }}
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+
+        <div class="my-450">
+            <h2>
+                Deprecated Spacing
+            </h2>
+            <p>
+                The following remain for backward compatibility but should not be used. Please refactor any code
+                that does use them, as they will be removed in a future version of ESDS.
+            </p>
+            <table class="table table-bordered">
+                <thead>
+                    <tr>
+                        <th scope="col">
+                            Class Name
+                        </th>
+                        <th scope="col">
+                            Multiplier (rem)
+                        </th>
+                        <th scope="col">
+                            Spacing (px)
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr
+                        v-for="space in deprecatedSpacers"
+                        :key="space.alias">
+                        <td>
+                            <code>{{ space.alias }}</code>
+                        </td>
+                        <td>
+                            {{ space.em }}
+                        </td>
+                        <td>
+                            {{ space.px }}
+                        </td>
+                    </tr>
+                </tbody>
```

Comment:
Seems like most of the old classes have a direct mapping to a new class. Could we provide a recommended "new" class to use when migrating?

Reply:
Good idea! Added.